pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/generator/SegmentGenerationAndPushTaskGenerator.java

```diff
@@ -220,15 +220,13 @@ public String getTaskType() {
     String pushMode = IngestionConfigUtils.getPushMode(batchConfigMap);
 
     Map<String, String> singleFileGenerationTaskConfig = new HashMap<>(batchConfigMap);
+    singleFileGenerationTaskConfig.put(BatchConfigProperties.TABLE_NAME,
```

Comment:
We might want to put the offline table name (table name with type) in case we want to support generating segments for realtime table in the future.

Reply:
change to put type as well